index.html

```diff
@@ -188,7 +211,7 @@ <h3 class="section-header">Currently in use at</h3>
     </div>
 </section>
 <section id="architecture">
-    <div class="section-grey">
+    <div class="section-white">
```

Comment:
Alternating grey and white. This requires making the logos a bit darker, hence the CSS change.